README.md

```diff
@@ -10,6 +10,8 @@ The project is a monorepo. Packages are located in the `packages` directory. The
 - `@netlify/remix-runtime` - The Remix runtime for Netlify Edge Functions
 - `@netlify/remix-edge-adapter` - The Remix adapter for Netlify Edge Functions
 - `remix-edge-demo-site` - A demo site for testing the packages
+- `@netlify/hydrogen-edge-adapter` - The Hydrogen/Remix adapter for Netlify Edge Functions
+- `@netlify/hydrogen-netlify-plugin` - The Netlify Build Plugin for Hydrogen projects on Netlify
```

Comment:
Wondering if we should name this `@netlify/hydrogen-netlify-integration` @netlify/ecosystem-pod-frameworks?

Reply:
I wonder if omittig `-netlify` makes sense as well as it's a scoped package `@netlify`, so `@netlify/hydrogen-integration`.

Thoughts @MarcL?

Reply:
I went ahead and renamed it to `@netlify/hydrogen-integration`. We can always put it back to `@netlify/hydrogen-netlify-integration`.

I also wonder if it's something like the Next runtime, so maybe `@netlify/hydrogen-runtime`?